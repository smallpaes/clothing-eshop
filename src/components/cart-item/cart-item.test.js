import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

it('Should render CartItem component', () => {
  const mockProps = { 
    imageUrl: 'www.test.com', 
    price: 10, 
    name: 'hats', 
    quantity: 2 
  };
  const wrapper = shallow(<CartItem item={mockProps} />);
  expect(wrapper).toMatchSnapshot();
});