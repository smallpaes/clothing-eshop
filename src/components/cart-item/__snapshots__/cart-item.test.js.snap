// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render CartItem component 1`] = `
<styled.div>
  <styled.img
    alt="item"
    src="www.test.com"
  />
  <styled.div>
    <span
      className="name"
    >
      hats
    </span>
    <span
      className="price"
    >
      2
       x $
      10
    </span>
  </styled.div>
</styled.div>
`;
